core/extension/native_extension.cpp

```diff
@@ -297,7 +297,7 @@ NativeExtension::InitializationLevel NativeExtension::get_minimum_library_initia
 }
 void NativeExtension::initialize_library(InitializationLevel p_level) {
 	ERR_FAIL_COND(library == nullptr);
-	ERR_FAIL_COND(p_level <= int32_t(level_initialized));
+	ERR_FAIL_COND_MSG(p_level <= int32_t(level_initialized), "Level '" + itos(p_level) + "' must be higher than the current level: '" + itos(level_initialized) + "'");
```

Comment:
You can use `vformat` to simplify:
```suggestion
	ERR_FAIL_COND_MSG(p_level <= int32_t(level_initialized),
			vformat("Level '%d' must be higher than the current level '%d'.", p_level, level_initialized));
```
(Didn't check if it would require explicit casting from enum to int but since `itos` worked without I think it should not be needed.)

Reply:
Far better. Thank you!